run_linux.go

```diff
@@ -605,11 +605,28 @@ func runMakeStdioPipe(uid, gid int) ([][]int, error) {
 }
 
 func setupNamespaces(logger *logrus.Logger, g *generate.Generator, namespaceOptions define.NamespaceOptions, idmapOptions define.IDMappingOptions, policy define.NetworkConfigurationPolicy) (configureNetwork bool, configureNetworks []string, configureUTS bool, err error) {
+	defaultContainerConfig, err := config.Default()
+	if err != nil {
+		return false, nil, false, fmt.Errorf("failed to get container config: %w", err)
+	}
+
 	// Set namespace options in the container configuration.
 	configureUserns := false
 	specifiedNetwork := false
 	for _, namespaceOption := range namespaceOptions {
 		switch namespaceOption.Name {
+		case string(specs.IPCNamespace):
+			if !namespaceOption.Host {
+				for _, sysctl := range defaultContainerConfig.Sysctls() {
+					splitn := strings.SplitN(sysctl, "=", 2)
```

Comment:
Not a blocker but: both processing sysctls for IPC and Network are using common parsing code there is a room to create a function in `internal/parse` called `parseSysctl(token)` and remove the redundant parsing code.